website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

```diff
@@ -0,0 +1,298 @@
+---
+title: "Lint and format your code"
+id: "lint-format"
+description: Integrate with popular linters and formatters like SQL Fluff, sqlfmt, Black, and Prettier."
+sidebar_label: "Lint and format"
+tags: [IDE]
+---
+
+Enhance your development workflow to focus on code quality by integrating with popular linters and formatters like [SQLFluff](https://sqlfluff.com/), [sqlfmt](http://sqlfmt.com/), [Black](https://black.readthedocs.io/en/latest/), and [Prettier](https://prettier.io/). Leverage these powerful tools directly in the dbt Cloud IDE without interrupting your development flow.
+
+<details>
+<summary>What are linters and formatters? </summary>
+Linters analyze code for errors, bugs, and style issues, while formatters fix style and formatting rules. 
+</details>
+
+
+In the dbt Cloud IDE, you have the capability to perform linting and formatting on five different file types (SQL, YAML, Markdown, JSON, and Python):
+ 
+- SQL &mdash; [Lint](#lint) with SQLFluff and [format](#format) with sqlfmt
```

Comment:
you can also `Fix` with sqlfluff!

Reply:
ah-mazing @dave-connors-3 ! i'm going to change this line to add 'fix' so appreciate your feedback 🙏